### PR TITLE
Refactor Related Work

### PR DESCRIPTION
**Pull request by jkrvivian**
* Combine similar works into one paragraph
* Eliminate statesments of IOTA (move to Chap 3)
* Rephrase

### CODE REVIEW

File: deprecated.tex [hunk 1]
```diff
@@ -5,7 +5,18 @@
...
...posed by IOTA Foundation, targets for IoT data streams trading. Data streams are stored via MAM and data payment is paid in IOTA. Nevertheless, the platform design is centralized, where new devices require manual approval to be visible in the marketplace. O. Lamtzidis and J. Gialelis \cite{IOTASensorNode} propose a distributed sensor node (SN) system to exchange data and establish a data monetization economy paradigm. Sensor nodes upload data to IOTA network via MAM protocol, and the Back-End server helps SNs to accelerate transaction attachments. The Back-End server also manages keys of data streams, record a tuple of SN, SN metadata, keys and available data stream entry points on MAM, and evaluates data price according to it's quality. Furthermore, as the Back-End server keeps all information of data products, it operates an user friendly interface of the marketplace and tackles all trading procedures. The heavy loading of Back-End server encounters the system scalability problem, single point failure as well as malicious attacks which may damage the profits of all players. And the pricing strategy is rather simple, which do not take subscription based diagram into consideration.
```
Comment (jserv) [inline deprecated.tex:14]: Why did you mention `SN`?

File: paper.tex [hunk 2]
```diff
@@ -100,23 +100,19 @@ \section{Related Work}
...
+A. Colman et al. \cite{TrustedMarketplaceWearable} focuses on trading the sensitive data among wearables. Several security issues are well discussed and solved with DLTs in the paper, the system consists multiple components that has its different responsibilities, such as data anonymizer eliminates keen information for privacy protection, access controller handles the key management and contract manager manages Ethereum smart contracts. While the trading processes are splited into multiple servers, and the request/response communication model is applied, the scalability problem of the marketplace remains.
```
Comment (jserv) [inline paper.tex]: s/focuses/discussed/

File: paper.tex [hunk 3]
```diff
@@ -107,7 +107,7 @@ \subsection{Data exchange mechanism for IoT}
...
+A. Colman et al. \cite{TrustedMarketplaceWearable} discussed on trading the sensitive data among wearables. Several security issues are well discussed and solved with DLTs in the paper, the system consists multiple components that has its different responsibilities, such as data anonymizer eliminates keen information for privacy protection, access controller handles the key management and contract manager manages Ethereum smart contracts. While the trading processes are splited into multiple servers, and the request/response communication model is applied, the scalability problem of the marketplace remains.
```
Comment (jserv) [inline paper.tex]: Should be "consists of"

File: paper.tex [hunk 4]
```diff
@@ -107,11 +107,13 @@ \subsection{Data exchange mechanism for IoT}
...
+
+O. Lamtzidis and J. Gialelis \cite{IOTASensorNode} proposed a distributed sensor node system to exchange data and establish a data monetization economy paradigm. The Back-End server helps sensor nodes to accelerate data uploads to MAM, manages keys and metadatas of data streams, and evaluates data price according to it's quality. Furthermore, the Back-End server operates an user friendly interface of the marketplace and tackles all trading procedures. The heavy loading of Back-End server encounters the system scalability problem, single point failure as well as malicious attacks which may damage the profits of all players. And the pricing strategy allows buyout, but does not take subscribe/unsubscribe based diagram into consideration. The decentralized data marketplace in \cite{DDMSmartCities} is fully decentralized which does not exist any intermediate server but data providers only, data providers attach and trade data streams via MAM. With MAM, the data streams subscription is enabled, yet the conditions and details like refund and unsubscription are not discussed in this paper.
```
Comment (jserv) [inline paper.tex]: should be "its quality."

Comment (jserv) [reply paper.tex]: use "backend."

Comment (jkrvivian) [reply paper.tex]: The authors name the backend server `Back-End` server

File: paper.tex [hunk 5]
```diff
@@ -113,7 +113,7 @@ \subsection{Trusted IoT Trading Infrastructure}
...
+O. Lamtzidis and J. Gialelis \cite{IOTASensorNode} proposed a distributed sensor node system to exchange data and establish a data monetization economy paradigm. The Back-End server helps sensor nodes to accelerate data uploads to MAM, manages keys and metadatas of data streams, and evaluates data price according to its quality. Furthermore, the Back-End server operates an user friendly interface of the marketplace and tackles all trading procedures. The heavy loading of Back-End server encounters the system scalability problem, single point failure as well as malicious attacks which may damage the profits of all players. And the pricing strategy allows buyout, but does not take subscribe/unsubscribe based diagram into consideration. The decentralized data marketplace in \cite{DDMSmartCities} is fully decentralized which does not exist any intermediate server but data providers only, data providers attach and trade data streams via MAM. With MAM, the data streams subscription is enabled, yet the conditions and details like refund and unsubscription are not discussed in this paper.
```
Comment (jserv) [inline paper.tex]: Fix the grammar error: "which does not exit"

Comment (jkrvivian) [reply paper.tex]: :thinking: I don't get it.

Comment (jserv) [reply paper.tex]: Change "The decentralized data marketplace in is fully decentralized which does not exist ..." to
> The decentralized data marketplace in is fully decentralized without the existence ..."

**Pull request merged**